query-service-impl/src/test/java/org/hypertrace/core/query/service/QueryRequestBuilderUtils.java

```diff
@@ -38,18 +39,31 @@ public static Filter createEqualsFilter(String column, String value) {
     return createFilter(column, Operator.EQ, createStringLiteralValueExpression(value));
   }
 
+  public static Filter createEqualsFilter(String column, long value) {
+    return createFilter(column, Operator.EQ, createLongLiteralValueExpression(value));
+  }
+
+  public static Filter createEqualsFilter(String column, int value) {
+    return createFilter(column, Operator.EQ, createIntLiteralValueExpression(value));
+  }
+
   public static Filter createFilter(String column, Operator operator, Expression expression) {
     return Filter.newBuilder().setLhs(createColumnExpression(column))
         .setOperator(operator).setRhs(expression).build();
   }
 
   public static Expression createStringLiteralValueExpression(String value) {
     return Expression.newBuilder().setLiteral(LiteralConstant.newBuilder().setValue(
-        Value.newBuilder().setString(value))).build();
+        Value.newBuilder().setString(value).setValueType(ValueType.STRING))).build();
```

Comment:
This was written long time ago and I don't remember if there was any reason. It's just that we got started there and continued that way AFAIK.

Reply:
Looking at value and trying to reach way back into the mists of the past, I believe it was because oneof doesn't support repeated and maybe map fields. A wrapper would also have sufficed there, but that was likely the reason.